src/app/analyze/stack/stack-recommendation/stack-recommendation.html

```diff
@@ -15,7 +15,7 @@
               <!-- TODO: To change the event to delegate instead of attaching more events -->
               <span *ngIf="!clickable">
                  <label>{{row.custom.name}}</label>
-                 <input name="{{row.name}}" [type]="row.custom.type" value="{{row.custom.name}}" />
+                 <input name="{{row.name}}" [type]="row.custom.type" value="{{row.custom.name}}" (change)="eventChangeHandler(row)"/>
```

Comment:
this is not built properly. The input name or ID should be in the lable "for" attribute so that they are linked and other systems can read the correct label with the input field.
https://www.w3schools.com/TAGs/tag_label.asp